src/test/java/org/opentripplanner/middleware/persistence/PersistenceUtil.java

```diff
@@ -77,4 +80,63 @@ public static void deleteTripRequests(List<TripRequest> tripRequests) {
         }
     }
 
+    public static MonitoredTrip createMonitoredTrip(String userId) {
+        Set<String> days = new HashSet<>();
+        days.add("Monday");
```

Comment:
Should the days lowercase or uppercase? See https://github.com/ibi-group/otp-middleware/blob/e85febf23a8accad01a28f49d26bcb63cc8053ad/src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java#L42

Reply:
I've changed to lowercase. I've also added a comment to the parameter in MonitoredTrip regarding case! Shame enums can't be used.